openedx/openedx-platform
# feat: add open_managed team type

## Description

Create a new open_managed team type. Ref: https://openedx.atlassian.net/wiki/spaces/COMM/pages/3885760525/Open+Managed+Group+Type

## Testing instructions

1. Clone `frontend-app-course-authoring` from the following PR https://github.com/openedx/frontend-app-course-authoring/pull/669
2. Mount the repo as editable in tutor: `tutor mounts add frontend-app-course-authoring`
3. Go to the CMS -> Pages and Resources -> Teams, and enable teams:
![image](https://github.com/openedx/edx-platform/assets/33465240/6ed2bd99-cdcb-43cd-b433-ef9191c2795a)
4. Create a new group of type `Open managed`
![image](https://github.com/openedx/edx-platform/assets/33465240/913bda18-31c1-49aa-b7cd-5a70ec2c2455)
5. Verify users can join and see other teams but cannot create new teams.
![image](https://github.com/openedx/edx-platform/assets/33465240/7bc3e2b1-0069-4a43-bd8f-108df42ce5b1)


## Deadline

None
